lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml

```
@@ -362,6 +362,9 @@ states:
       enhanced-verification:
         targetJourney: FAILED
         targetState: FAILED
+      fraud-fail-with-no-ci:
+        targetJourney: FAILED
+        targetState: FAILED
```

Can a user still enter an international address if they visit address CRI without the `internationalAddress` context?

We should probably continue to route them to `EVALUATE_SCORES` and let that fail them if they are not eligible.

We'll want to do that when we introduce the fraud resilience work anyway - since it could happen on any journey.